getting_started/getting_started.org

```diff
@@ -0,0 +1,389 @@
+  that hard, right? It seems that a configuration instance needs to
+  define a single key, called =valuesyaml= that needs to be an
+  object. I think we can do that! Let's actually start benefiting from
+  our efforts and create our first Pligos configuration.
+
+* hello-world Pligos configuration
+
+  As a helm plugin, every Pligos configuration naturally is also a
+  helm chart. I think we already mastered creating helm charts so this
+  one should be easy:
+
+  #+begin_src sh
+    helm create hello-world
+  #+end_src
+
+  Templates will be provided by our Flavor, so let's kill them with fire!
+
+  #+begin_src sh
+    rm -r hello-world/templates
+  #+end_src
+
+  In order to morph this beautiful helm chart to a Pligos capable one,
+  I propose to rename the =values.yaml= to something more useful:
+
+  #+begin_src sh
+    mv hello-world/values.yaml hello-world/pligos.yaml
+  #+end_src
+
+  =values.yaml= and our beloved =templates= will come back soon, but,
+  as they are generated,  we don't actually need to commit them
+  anymore, so I think it's good practice to ignore them:
+
+  #+begin_src sh
+    printf "/values.yaml\n/templates\n" > hello-world/.gitignore
+  #+end_src
+
+  Let's start editing the =pligos.yaml=. At first we are going to add
+  a header to the file that defines some metadata:
+
+  #+begin_src yaml
+    pligos:
+      version: '1'
+      types: [../types.yaml]
+  #+end_src
+
+  You might notice, that we can actually plug multiple type definition
+  files into your Pligos configuration. Multiple files are simply
+  going to be merged.
+
+  We are actually getting really close to something that can be
+  deployed, so bare with me, we can do this! The only thing left is
+  creating a /Context/.
+
+* Contexts
+
+  Contexts allow you to manage different versions of your service
+  configurations. The most obvious use case is to have a development
+  and production version. Doing this without Pligos normally requires
+  managing multiple =values.yaml= files, each one overwriting a base
+  =values.yaml= file. Using Pligos this nonsense can finally come to
+  an end because you can manage all the environments side by side
+  inside of one file. You will see how this scales (spoiler: this is
+  what type instances are for).
+
+  To ease into the concept let's create a single first context called
+  =default= in our =schema.yaml=:
+
+  #+begin_src yaml
+    contexts:
+      default:
+        flavor: ../webservice
+        spec:
+          values:
+            valuesyaml:
+              image:
+                repository: nginx
+                tag: stable
+                pullPolicy: IfNotPresent
+
+              service:
+                type: ClusterIP
+                port: 80
+
+              ingress: {enabled: false}
+  #+end_src
+
+  Go ahead, add this context configuration under the header and remove
+  the rest of the helm =values.yaml= defaults.
+
+  Looks weird? Good, because that's not how you would normally use
```

Comment:
Yellow flag. It does look weird. Maybe because I "know" how to use pligos I was confused. Maybe teaching something wrong to smooth the learning steps can be good, but could also confuse the starter. I'll put the text to test soon.